content/blog/2022/09/2022-09-18-digital-ocean-sponsorship.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Digital Ocean and Jenkins partnership continues to grow!"
+tags:
+- jenkins
+- digitalocean
+- community
+author: kmartens27
+description: "DigitalOcean increases Jenkins sponsorship"
+opengraph:
+  image: /images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png
+---
+
+We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
+Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
+image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]
+
+DigitalOcean has provided a substantial donation of $18,000 to the Jenkins project.
+This is in addition to their previous sponsorship link:https://www.jenkins.io/blog/2022/05/04/DigitalOcean/[described here]. 
+Combining these, DigitalOcean has granted over *$20,000* to the Jenkins project.
+The funding aspect is clear, but there are several other benefits to working with DigitalOcean in this capacity.
+Jenkins has access to more resources than ever before, such as utilizing the DigitalOcean platform to continue development and share progress made.
+We can dedicate our time and resources more fully, allowing users to engage and make an impact.
+
+This sponsorship also increases the impact that we can have as a community, when it comes to ongoing projects and events like Hacktoberfest, which DigitalOcean organizes and hosts.
+DigitalOcean is proving their commitment to open source, which matches our desire to improve Jenkins in any way and by any means.
+
+If you are new to link:https://digitalocean.com/[DigitalOcean] and would like to learn more, use our promo code `DOJENKINS100` to get $100 credit to use over your first 60 days. 
+This offer is only valid for new accounts. 
```

Comment:
(nitpicking suggestion)
The final phrase could be put between "(", as a side note.

Reply:
definitely agree here @jmMeessen , I've gone ahead and added it to the end of the previous sentence and surrounded it with parentheses.